.sparkswap/config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Sparkswap configuration for use inside the container. (Note the rpcCertPath is relative to being inside the container)
+ *
+ * It will use the user defined rpc address, user and password if they exist, otherwise
+ * they will use the default.
+ */
+
+module.exports = {
+  /**
+   * Address of the host for the Broker Daemon gRPC Server
+   * @type {String}
+   */
+  rpcAddress: process.env.RPC_ADDRESS || 'localhost:27492',
+
+  /**
+   * Default path of the Broker Daemons RPC Public Cert
+   * @type {String}
+   */
+  rpcCertPath: '/secure/broker-rpc-tls.cert',
```

Comment:
Worth adding a comment that it will always be here because this is where it is mounted in the container.